### PR TITLE
test: remove setup helper

### PR DESCRIPTION
**Pull request by danielgent**
Going through comments and I realised that the fixtures in this helper `await setup(worker)` are never used!

May as well keep the tests simpler by removing it

### CODE REVIEW

File: integration-test/flows/code-flow.spec.ts [hunk 1]
```diff
@@ -19,8 +18,6 @@ describe("code-flow", () => {
   });
 
   it("should run a passwordless flow with code", async () => {
-    await setup(worker);
```
Comment (danielgent) [inline integration-test/flows/code-flow.spec.ts:22]: lots of this kind of thing. Tests still pass!

File: integration-test/helpers/setup.ts [hunk 2]
```diff
@@ -1,77 +0,0 @@
...
-      authorization: `Bearer ${token}`,
-      "content-type": "application/json",
-    },
-  });
-
-  if (createTenantResponse.status !== 201) {
-    throw new Error(
-      `Setup: Failed to create tenant: ${await createTenantResponse.text()}`,
-    );
-  }
```
Comment (danielgent) [inline integration-test/helpers/setup.ts:24]: we already have tests for adding & removing tenants in `integration-test/management-api/tenants.spec.ts`

File: integration-test/helpers/setup.ts [hunk 3]
```diff
@@ -1,77 +0,0 @@
...
-        allowed_logout_urls: "",
-        email_validation: "disabled",
-        client_secret: "appSecret",
-      }),
-      headers: {
-        authorization: `Bearer ${token}`,
-        "content-type": "application/json",
-      },
-    },
-  );
```
Comment (danielgent) [inline integration-test/helpers/setup.ts:68]: We don't have any tests for this though. Adding integration tests for all our management API endpoints is one of the last things we should do...

Maybe I should do this before merging this :thinking:

File: integration-test/helpers/setup.ts
Comment (danielgent) [inline integration-test/helpers/setup.ts:1]: we could bring this back if we actually use it!

File: integration-test/management-api/tenants.spec.ts [hunk 4]
```diff
@@ -60,25 +59,20 @@ describe("tenants", () => {
...
 
-    // two tenants in initial setup
     expect(tenantsResponse1.status).toBe(200);
     const body1 = (await tenantsResponse1.json()) as Tenant[];
-    expect(body1.length).toEqual(5);
-
-    // remove 'otherTenant'
-    const otherTenant = body1.find((t) => t.name === "otherTenant");
+    // base tenant + two tenants in test-server
+    expect(body1.length).toEqual(3);
```
Comment (danielgent) [inline integration-test/management-api/tenants.spec.ts:72]: this is the only change to the tests

Comment (danielgent) [reply integration-test/management-api/tenants.spec.ts:72]: we're no longer creating two extra tenants

**Review by markusahlstrand — APPROVED** [review]
Cool, less is more

**Pull request merged**